### DISCUSSION TITLE
Mapbox Pricing

### DISCUSSION THREAD

**Discussion by dhambleton** [code4recovery/12-step-meeting-list#1407] [Q&A]
I'm redesigning our local AA website and discovered that Mapbox is a requirement for the new TSML UI. I'm not familiar with the mapbox solution, but was able to get it set up, integrated, and working correctly. My main question is about the API requests and pricing. My interpretation is that all requests from the 12 step meeting plugin would fall under "Mapbox GL JS" map loads for web. This particular utilization of the API comes with 50,000 free page loads and then $5.00 per 1000 page loads after that. Am I correct in my understanding of the cost and utilization of the Mapbox API for the 12 step meeting plugin? I apologize if this is already documented elsewhere. I couldn't find it.

**Comment by joshreisner** [comment]
yes, that's right - here's some more context about the fees: https://github.com/code4recovery/12-step-meeting-list/discussions/1347

the tldr is we haven't heard of a site yet run into any fees, and mapbox's pricing is slightly better than google, last time i checked
